connector/spanmetricsconnector/connector.go

```diff
@@ -47,12 +48,6 @@ const (
 	metricNameCalls   = "calls"
 )
 
-var defaultLatencyHistogramBucketsMs = []float64{
-	2, 4, 6, 8, 10, 50, 100, 200, 400, 800, 1000, 1400, 2000, 5000, 10_000, 15_000,
-}
-
-type metricKey string
```

Comment:
What's the reason for removing the `metricKey` type definition? I feel it has the benefit of improving readability by communicating its intent to the reader, while also acting as a compile-time guard against using any old string.